### DISCUSSION TITLE
Moving quarkus update away from experimental?

### DISCUSSION THREAD

**Discussion by mocenas** [quarkusio/quarkus#43897] [Community]
Updating quarkus app using CLI and `quarkus update` goal is considered experimental. While the update mechanism is there for some time. 

Is there a plan to move this goal away from experimental status?

**Comment by rsvoboda** [comment]
```
[WARNING] quarkus:update goal is experimental, its options and output might change in future versions
```
When running `quarkus update` I see the above warning, @aloubyansky / @gsmet any plans to look more into update functionalities?
